src/views/tenant/standards/ApplyStandard.js

```diff
@@ -63,6 +63,14 @@ const ApplyStandard = () => {
         title="Tenant Choice"
         description="Choose the tenants to create the standard for."
       >
+        <CCallout color="danger">
+          Ensure you read{' '}
+          <a href="https://cipp.app/docs/user/usingcipp/tenantadministration/standards/#meet-the-standards">
+            the documentation fully
+          </a>{' '}
+          before proceeding with this wizard, noting that some changes here are not reversible
```

Comment:
change to "before proceeding with this wizard. Some of the changes cannot be reverted by CIPP"

Reply:
Amended~

Reply:
Amended~ @KelvinTegelaar